diff --git a/node_modules/mdast-util-mdx-expression/lib/index.js b/node_modules/mdast-util-mdx-expression/lib/index.js
index 53f09cb..d78de64 100644
--- a/node_modules/mdast-util-mdx-expression/lib/index.js
+++ b/node_modules/mdast-util-mdx-expression/lib/index.js
@@ -80,7 +80,13 @@ function exitMdxExpression(token) {
   const node = this.stack[this.stack.length - 1]
   assert(node.type === 'mdxFlowExpression' || node.type === 'mdxTextExpression')
   this.exit(token)
-  node.value = value
+
+  if (
+    token.estree?.type === "Program" && value.trim()[0] !== "{"
+  ) {
+    node.value = `{${value}}`
+  } else {
+    node.value = value
 
   if (estree) {
     node.data = {estree}
