diff --git a/node_modules/rehype-minify-whitespace/index.js b/node_modules/rehype-minify-whitespace/index.js
index f1412ac..6560bc8 100644
--- a/node_modules/rehype-minify-whitespace/index.js
+++ b/node_modules/rehype-minify-whitespace/index.js
@@ -161,6 +161,8 @@ function all(parent, context) {
   let length = children.length
   let index = -1
 
+  context.whitespace = parent.tagName === 'code' ? 'pre' : context.whitespace
+
   while (++index < length) {
     const result = minify(
       children[index],
